server/service/MailService.ts

```diff
@@ -240,7 +249,7 @@ const service = {
 
     const toUserId = relatedComment ? relatedComment.userId : flight.userId;
```

Comment:
ich bekomme bei "relatedComment.userId" die Fehlermeldung "Property 'userId' does not exist on type 'UserInstance'" angezeigt.

Aber es ist an der Stelle ja auch eine FlightCommentInstance.